example/src/components/DropDownExamples.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import dropdown from '../../../src/components/DropDown';
+import Code from '../../../src/components/Code';
+import { Icon, IconTypes } from '../../../src/components/Icon';
+
+export default function DropDownExamples() {
+  return (
+    <section id="DropDownExamples">
+      <h2 className="base--h2">Drop Down</h2>
+      <div className="row">
+
+      <div className="block-example">
+      <div className="dropdown">
+        <div className="dropbtn">Dropdown <Icon className="drowndownicon" type="right" size="small" /></div>
```

Comment:
The Icon classname should also be regularized with the rest of the classnames.  I don't know what a 'drown' is or why it relates to a downicon.